test/e2e/run_test.go

```diff
@@ -213,7 +229,7 @@ var _ = Describe("Podman run", func() {
 
 	})
 
-	It("podman run --tls-verify", func() {
+	FIt("podman run --tls-verify", func() {
```

Comment:
this skips all tests except this one and does rightfully marks the tests as fail so something like this is never merged by accident

Reply:
Yeah sorry!! I thought I reverted all of those!